build/common.props

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <LangVersion>latest</LangVersion>
     <MajorVersion>4</MajorVersion>
-    <MinorVersion>$(MinorVersionPrefix)35</MinorVersion>
+    <MinorVersion>$(MinorVersionPrefix)36</MinorVersion>
     <PatchVersion>4</PatchVersion>
```

Comment:
Shouldn't we set the patch version to 0?

Reply:
@kshyju  I overlooked the patch version, so thank you for pointing it out. Your keen observation is greatly appreciated. To address this, I've added another commit. Thank you.